README.md

```diff
@@ -16,10 +16,9 @@
 
 <div align="center">
 
-[![Crate Badge]][Crate] [![Docs Badge]][API Docs] [![CI Badge]][CI Workflow] [![Deps.rs
-Badge]][Deps.rs]<br> [![Codecov Badge]][Codecov] [![License Badge]](./LICENSE) [![Sponsors
-Badge]][GitHub Sponsors]<br>
-[Ratatui Website] · [API Docs] · [Widget Examples] · [App Examples] · [Changelog]<br>
+[![Crate Badge]][Crate] [![Repo Badge]][Repo] [![Docs Badge]][Docs] [![License Badge]][License]  \
+[![CI Badge]][CI] [![Deps Badge]][Deps] [![Codecov Badge]][Codecov] [![Sponsors Badge]][Sponsors]  \
+[Ratatui Website] · [Docs] · [Widget Examples] · [App Examples] · [Changelog]  \
```

Comment:
<img width="642" alt="image" src="https://github.com/user-attachments/assets/312137a2-e89c-40f5-ba98-52d9c557c9d8" />

Adds a repo link to the badges header, which is nice because the repo link is always below the fold on crates.io.
Makes the docs badge static - it's rare that this fails